tests/queries/0_stateless/02771_log_faminy_truncate_count.sql

```diff
@@ -1,3 +1,4 @@
+-- Tags: no-parallel
 DROP TABLE IF EXISTS default.test_log;
 CREATE TABLE default.test_log
```

Comment:
Can we just remove both `default` and `no-parallel`? `clickhouse-test` runs each test in a separate database to allow parallel execution